CyborgTests/XMLSchemaTests.swift

```diff
@@ -110,4 +110,44 @@ class XMLSchemaTests: XCTestCase {
         expected.closeSubpath()
         XCTAssertEqual(expected.copy()!, pathLayer.path!)
     }
+    
+    func test_two_nested_groups() {
+        let drawable = VectorDrawable
+            .create(from: """
+                <?xml version="1.0" encoding="utf-8"?>
+                <vector xmlns:android="http://schemas.android.com/apk/res/android"
+                android:viewportWidth="24"
+                android:viewportHeight="24"
+                android:width="24dp"
+                android:height="24dp">
+                <group
+                android:translateX="-3589"
+                android:translateY="-2800">
+                <group
```

Comment:
nit: indent the nested group a bit to make the structure more apparent

Reply:
Done.